ui/package.json

```diff
@@ -4,13 +4,13 @@
   "dependencies": {
     "jenkins-js-extp": "0.0.2",
     "jenkins-js-modules": "^1.5.0",
-    "jquery-detached": "^2.1.4-v4",
+    "jquery": "^3.5.1",
     "moment": "~2.8.3",
     "window-handle": "^1.0.0"
   },
   "scripts": {
     "mvnbuild": "gulp bundle",
-    "mvntest": "gulp test"
+    "mvntest": "jest --ci --config=jest.config.json --reporters=jest-standard-reporter --reporters=jest-junit"
```

Comment:
doesn't look like this will actually show up in CI by the looks of it as it's not archived, but I think the github action build would fail if the tests fail

Reply:
Maybe, I'm not sure if it will be archived, but it would definitely break CI.

Reply:
would be good to see it working, i can see the jasmine ones used to work, could you push a test failure?

Reply:
Sure

Reply:
@timja I have triggered a failure, please tell me once you review 👍